components/shared/CommonTokenMoney.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <TokenMoney
+    v-if="tokenId"
```

Comment:
tokenId isnt prop?

Reply:
tokenId is from `PrefixMixin`  rather than the props

Reply:
tokenId is props now